dm/syncer/err-operator/operator.go

```diff
@@ -94,6 +103,59 @@ func (h *Holder) Set(pos string, op pb.ErrorOp, events []*replication.BinlogEven
 	return nil
 }
 
+// GetBehindCommands gets behind commands.
```

Comment:
behind means earlier or later?

Reply:
it means later, earlier may be removed by `RemoveOutdated`.